src/app.config.ts

```diff
@@ -29,9 +29,6 @@ export default defineAppConfig({
         }
       }
     },
-    button: {
-      padding: 'p-0'
-    },
```

Comment:
This was only in here for a case used in bor-ui (which is the styling this project started with). It can be applied by using the `padded=false` prop if we need it on anything, but generally we will always want padding on our buttons I think